frontend/apps/marketing/src/components/contentful/faqAccordion/FAQAccordion.tsx

```diff
@@ -83,7 +83,33 @@ const FAQAccordionContentful: React.FunctionComponent<
   }
 
   return (
-    <FAQAccordion className={moduleStyles.faqAccordion} items={faqItems} />
+    <div>
```

Comment:
Nit: use the [Box component](https://mui.com/material-ui/react-box/) here for consistency w/ other MUI components.